tests/tools/archive/orm/test_links.py

```diff
@@ -262,6 +264,7 @@ def test_complex_workflow_graph_links(aiida_profile_clean, tmp_path, aiida_local
     assert set(export_set) == set(import_set)
 
 
+@pytest.mark.nightly
 def test_complex_workflow_graph_export_sets(aiida_profile, tmp_path, aiida_localhost_factory):
```

Comment:
Although not failed with pytest-xdist, I think this kind of DB test should run with `aiida_profile_clean` to start from a clean profile. If it can be moved to nightly then the slow down because of `aiida_profile_clean` is less noticeable.